code/datums/gamemode/factions/bloodcult/bloodcult_rituals.dm

```diff
@@ -358,14 +225,430 @@ var/global/list/cult_altars = list()       // List of cult altars in the world.
+	hit_targets = list()
+
+/datum/bloodcult_ritual/harm_crew/update_desc()
+	desc = "wield cult weaponry...<br>spill their blood...<br>sear their skin...<br>at least [targets - hit_targets] different individuals..."
+
+/datum/bloodcult_ritual/harm_crew/key_found(var/mob/living/L)
+	if (iscultist(L))
+		return FALSE
+	if (L in hit_targets)
+		return FALSE
+	hit_targets += L
+	if(hit_targets >= targets)
+		return TRUE
+	return FALSE
+
+////////////////////////SACRIFICE/////////////////////////////
+
+/datum/bloodcult_ritual/sacrifice_mouse
+	name = "Sacrifice Mouse"
+	desc = "a rodent...<br>an altar...<br>and a proper blade..."
+
+	ritual_type = "sacrifice"
+	difficulty = "easy"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list("altar_sacrifice_animal")
+
+/datum/bloodcult_ritual/sacrifice_mouse/key_found(var/mob/living/simple_animal/mouse/extra)
+	if(istype(extra))
+		return TRUE
+	return FALSE
+
+
+//////////////////////////////////////////////////////////////
+
+/datum/bloodcult_ritual/sacrifice_monkey
+	name = "Sacrifice Monkey"
+	desc = "a simian...<br>an altar...<br>and a proper blade..."
+
+	ritual_type = "sacrifice"
+	difficulty = "easy"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	keys = list("altar_sacrifice_monkey")
+
+/datum/bloodcult_ritual/sacrifice_monkey/key_found(var/extra)
+	return TRUE
+
+
+////////////////////////ALTAR/////////////////////////////////
+
+/datum/bloodcult_ritual/altar
+	name = "Prepare Altar"
+	desc = "raise an altar...<br>add proper paraphernalia around...<br>then plant a ritual knife on top..."
+
+	ritual_type = "altar"
+	difficulty = "easy"
+	personal = TRUE
+	reward_achiever = 200
+	reward_faction = 2
+
+	var/required_candles = 0
+	var/required_tomes = 0
+	var/required_runes = 0
+	var/required_pylons = 0
+	var/required_animal = 0
+	var/required_humanoid = 0
+	var/required_cultblade = 0
+
+	keys = list("altar_plant")
+
+/datum/bloodcult_ritual/altar/key_found(var/obj/structure/cult/altar/altar)
+	var/mob/user = owner.antag.current
+
+	var/valid = TRUE
+	var/found_candles = 0
+	for (var/obj/item/candle/blood/CB in range(1, altar))
+		if (CB.lit)
+			found_candles++
+	if (found_candles < required_candles)
+		to_chat(user, "<span class='sinister'>Need more lit blood candles...</span>")
+		valid = FALSE
+
+	var/found_tomes = 0
+	for (var/obj/item/weapon/tome/T in range(1, altar))
+		found_tomes++
+	if (found_tomes < required_tomes)
+		to_chat(user, "<span class='sinister'>Need more arcane tomes...</span>")
+		valid = FALSE
+
+	var/found_runes = 0
+	for (var/obj/effect/rune/R in range(1, altar))
+		found_runes++
+	if (found_runes < required_runes)
+		to_chat(user, "<span class='sinister'>Need more runes...</span>")
+		valid = FALSE
+
+	var/found_pylons = 0
+	for (var/obj/structure/cult/pylon/P in range(1, altar))
+		found_pylons++
+	if (found_pylons < required_pylons)
+		to_chat(user, "<span class='sinister'>You must construct additional pylons...</span>")
```

Comment:
small chance to play the audio from starcraft I please?

Reply:
I'd rather keep the reference subtle.

Reply:
damn